g2p_registry_rest_api/services/group.py

```diff
@@ -110,7 +119,8 @@ def createGroup(self, group_info):
         # TODO: create the group object
         logging.info("GROUP:")
 
-        grp_rec = self._process_group(group_info)
+        grp_helper = self.env["registrant_group.rest.service.helper"]
```

Comment:
It would be better to override `_process_group` to do this.

Reply:
Sorry didnt get you . Linking my previous comment [here](https://github.com/OpenG2P/openg2p-registry/pull/36#discussion_r1044578071)

Reply:
This is not the way it is usually done in Odoo. Take a look at using Mixin, that might be what you are trying to achieve.